### ISSUE TITLE
Request for Dataset

### ISSUE THREAD

**Issue by Jia-py** [yanweiyue/masrouter#11]
Awesome work!
Could you please share the dataset or the download link? Thank you very much!

**Comment by summervvind** [comment]
Thank you for sharing your dataset！
The script you provided appears to be for generating sample_gsm8k.jsonl, but I have the same question about splitting the dataset. For example, the following splitting of gsm8k:
```python
    train_dataset, test_dataset = split_list(dataset, 0.2)
    current_time = time.strftime("%Y-%m-%d-%H-%M-%S", time.localtime())
    log_file = f"gsm8k_{current_time}.txt"
    fix_random_seed(1234)
```

The `fix_random_seed()` is after the `split_list()`, but the `split_list()` calls `random.shuffle()`. This looks like there is no seed given in `split_list()`. Could you provide the seed or directly provide the split train and test dataset for gsm8k and humaneval?

Thank you very much!

**Comment by yanweiyue** [comment]
Thank you very much for raising this question. You wisely pointed out a serious issue in our code. The random seed should be set before the dataset is divided. The previous code setting resulted in unstable experimental results. We have moved the function that sets the random seed(1234) to the beginning of the code. However, this may cause the experimental results to differ from the initial reported results. We will conduct further experiments to ensure the reliability and stability of the results. Thank you very much for your feedback. We apologize for the many omissions in our code.
